third_party/proto/cosmos/crisis/v1beta1/genesis.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package cosmos.crisis.v1beta1;
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/crisis/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// GenesisState defines the crisis module's genesis state.
+message GenesisState {
+  // constant_fee is the fee used to verify the invariant in the crisis
+  // module.
+  cosmos.base.v1beta1.Coin constant_fee = 3 [
```

Comment:
is this supposed to start at 1?

Reply:
yup indeed, but this is from the SDK so we need to keep it like that